app/domain/logs.rb

```diff
@@ -290,6 +290,55 @@ module AuthnJwt
         code: "CONJ00056D"
       )
 
+      JWTAuthenticatorEntryPoint = ::Util::TrackableLogMessageClass.new(
```

Comment:
Hey @shulifink 
Can you please review this messages

Hey @sashaCher  can you review from security perspective

CC : @hilagross